lib/sequent/sequent.rb

```diff
@@ -1,3 +1,7 @@
+require 'oj'
+Oj.default_options={mode: :compat}
```

Comment:
Strange place for this configuration option? Can't we instantiate a global `sequent_oj` and use that everywhere?